app/assets/stylesheets/overlays/_edit_task_types_overlay.scss

```diff
@@ -14,6 +6,26 @@
       color: $tahi-blue-dark;
     }
   }
+
+  .task-headers h3 {
+    display: inline-block;
+
+    &:nth-child(2) {
+      float: right;
+    }
+  }
+
+  .individual-task-type {
+    margin-top: 20px;
+
+    p, div {
```

Comment:
Selector should have depth of applicability no greater than 2, but was 3<br>Each selector in a comma sequence should be on its own line